Lib/test/test_gc.py

```diff
@@ -766,6 +766,37 @@ def test_freeze(self):
         gc.unfreeze()
         self.assertEqual(gc.get_freeze_count(), 0)
 
+    def test_get_objects(self):
+        gc.collect()
+        l = []
+        l.append(l)
+        self.assertIn(l, gc.get_objects(generation=0))
+        self.assertNotIn(l, gc.get_objects(generation=1))
+        self.assertNotIn(l, gc.get_objects(generation=2))
+        gc.collect(generation=0)
+        self.assertNotIn(l, gc.get_objects(generation=0))
+        self.assertIn(l, gc.get_objects(generation=1))
+        self.assertNotIn(l, gc.get_objects(generation=2))
+        gc.collect(generation=1)
+        self.assertNotIn(l, gc.get_objects(generation=0))
+        self.assertNotIn(l, gc.get_objects(generation=1))
+        self.assertIn(l, gc.get_objects(generation=2))
+        gc.collect(generation=2)
+        self.assertNotIn(l, gc.get_objects(generation=0))
+        self.assertNotIn(l, gc.get_objects(generation=1))
+        self.assertIn(l, gc.get_objects(generation=2))
+        del l
+
+        # Check argument values
```

Comment:
Sometimes adding a comment means that it's time to split your function. Maybe split the function to create "def test_get_objects_arg(self)"?

Reply:
Done in 5b62014